content/participants/ttjc@ishita303/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Ishita Jain
+type: "participant"
+name: "Ishita Jain"
+portfolio: "https://ishitajain.com"
+github-link: "https://github.com/ishita303"
+linkedin-link: "https://www.linkedin.com/in/ishita-jain-459075175/"
+looking-for: "internship"
+description: "Website Developer|Initiator at WUFU"
+---
+
+Hi! I am Ishita Jain let me walk you through my intro, projects and blogs.
+
+### introduction
+
+I am a tech enthusiast, well conversant with languages like HTml, Css, JavaScript, PhP, ReactJs, Nodejs, Perl, C, C++ and Python. Also, comfortable with various CMS Platforms.
+
+## why are you taking this challenge?
+
+I am looking forward to an exposure. Learning something new and also aiming at grabbing the internship opportunity.
+
+## something interesting about you?
+
+I sing and play guitar. I started blogging recently, also started with a novel.
+
+### help I need?
+
+I would love to learn more about APi's and deploying applications.
+
+### how to contact me?
+
+- Telegram
```

Comment:
can you mention links to your profile

Reply:
My portfolio is not live yet, you can view over linkedin- https://www.linkedin.com/in/ishita-jain-459075175/

Reply:
From that, I mean links for contact as you mentioned telegram and LinkedIn.
I will add your LinkedIn profile in the contact me section 😎

Reply:
Sure, Linkedin- https://www.linkedin.com/in/ishita-jain-459075175/